docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -238,11 +241,10 @@ feedback.attachTo(document.querySelector("#your-button"), {
 Alternatively, you can call `feedback.createForm()` and have full control over when the form gets loaded, added to the dom, and finally shown to the user. Here is an example using JSX to define a button:
 
 ```typescript
-import { BrowserClient, Feedback, getClient } from "@sentry/react";
+import { getFeedback } from "@sentry/react";
```

Comment:
Not sure but should this be javascript instead of react here?

Reply:
```suggestion
import { getFeedback } from "@sentry/browser";
```

Reply:
In this case, the example is using JSX react intentionally. 

I mentioned it in the paragraph above, and the function in the same snippet uses `<button>` which is a react thing (well, it's a jsx thing which some other frameworks also use, but react started it)